src/getTime.js

```diff
@@ -18,7 +18,8 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  let timeMatches = str.match(/((\b[01]{1}\d{1})|(2{1}[0-3]{1})):[0-5]\d{1}\b/);
```

Comment:
[prefer-const](https://mate-academy.github.io/style-guides/javascript.html#references--prefer-const)

Reply:
done

Reply:
It is still a let variable
Also, you do not need `{1}` - without counter means `{1}`

Reply:
done, was not pushed to repo